subprojects/packagefiles/luajit/meson.build

```diff
@@ -121,6 +121,8 @@ DASM_FLAGS += ['-D', f'VER=@LJ_ARCH_VERSION@']
 
 if host_machine.system() == 'darwin'
   unwind_external = true
+elif host_machine.system() == 'windows'
```

Comment:
Shouldn't this be `cc.get_id() == 'msvc'` to mimic original `msvcbuild.bat`. Otherwise we can use it. Or in fact we could make error of running `./unwind_check.sh` not critical and assume that `unwind_external` is false in this case, with a warning.

Reply:
Good point, just matching `msvcbuild.bat` is not enough here, but I now also saw that the [Makefile](https://github.com/LuaJIT/LuaJIT/blob/f73e649a954b599fc184726c376476e7a5c439ca/src/Makefile#L311-L315) also disables `LUAJIT_UNWIND_EXTERNAL` whenever the target is Windows (no matter the compiler).

Reply:
Seems to be good then.